README.md

```diff
@@ -414,6 +415,10 @@ Table of Contents
   * [sentry.io](https://sentry.io/) — Sentry tracks app exceptions in real-time, has a small free plan. Free for 5k errors per month/ 1 user, unrestricted use if self-hosted
   * [honeybadger.io](https://www.honeybadger.io) - Exception, uptime, and cron monitoring that's so awesome, you'll wish your site had more errors. Honeybadger is free for solo devs and open-source projects (12,000 errors/month).
 
+## Chaos Engineering
+
+  * [Gremlin](https://www.gremlin.com/gremlin-free-software) — Gremlin's Chaos Engineering tools allow you to safely, securely, and simply inject failure into your systems to find weaknesses before they cause customer-facing issues.
```

Comment:
please add a short description of whats free, it would need to go into another section as we dont do single entry sections, list already to big for that kind of thing

Reply:
@ripienaar let me know what section you think this works in. we could also add a few more FOSS Chaos Engineering tools from https://github.com/dastergon/awesome-chaos-engineering

Reply:
adding more would be good too as long as they are saas

Reply:
@ripienaar I believe Gremlin is the only free SaaS offering for Chaos Engineering, so let me know what subheading would work and I'll move!

Reply:
probably not a perfect fit, but `testing` seems about nearest in that case